tests/Behat/features/multiple-wysiwyg-configs.feature

```diff
@@ -18,23 +18,23 @@ Feature: multiple elemental blocks with different HTMLEditorConfig instances
     Given I see a list of blocks
     # Check the link menu in the WYSIWYG "insert link" button is correct for block 1
     When I click on block 1
-    And I click on the "#Form_ElementForm_1 [aria-label^='Insert link'] button" element
-    Then I should see "Page on this site" in the ".mce-menu.mce-in" element
-    And I should see "Link to a file" in the ".mce-menu.mce-in" element
-    And I should see "Link to external URL" in the ".mce-menu.mce-in" element
-    And I should see "Anchor on a page" in the ".mce-menu.mce-in" element
-    And I should see "Link to email address" in the ".mce-menu.mce-in" element
+    And I click on the "#Form_ElementForm_1 button[aria-label^='Insert link']" element
+    Then I should see "Page on this site" in the ".tox-menu" element
+    And I should see "Link to a file" in the ".tox-menu" element
+    And I should see "Link to external URL" in the ".tox-menu" element
+    And I should see "Anchor on a page" in the ".tox-menu" element
+    And I should see "Link to email address" in the ".tox-menu" element
     # Check the link menu in the WYSIWYG "insert link" button is correct for block 2
     When I click on block 2
     # In CI, the mouse position just happens to produce a tooltip that stops us clicking on the insert link button
     # so we have to move the mouse somewhere else to avoid that
     And I click on the "input[type='text']" element
-    And I click on the "#Form_ElementForm_2 [aria-label^='Insert link'] button" element
-    Then I should see "Page on this site" in the ".mce-menu.mce-in" element
-    And I should see "Link to a file" in the ".mce-menu.mce-in" element
-    And I should see "Link to external URL" in the ".mce-menu.mce-in" element
-    And I should see "Anchor on a page" in the ".mce-menu.mce-in" element
-    And I should see "Link to email address" in the ".mce-menu.mce-in" element
+    And I click on the "#Form_ElementForm_2 button[aria-label^='Insert link']" element
+    Then I should see "Page on this site" in the ".tox-menu" element
+    And I should see "Link to a file" in the ".tox-menu" element
+    And I should see "Link to external URL" in the ".tox-menu" element
+    And I should see "Anchor on a page" in the ".tox-menu" element
+    And I should see "Link to email address" in the ".tox-menu" element
```

Comment:
These changes probably need to be on 5.1 - I suspect that branch's CI will be failing without it.

Reply:
Without these changes I get ` Element #Form_ElementForm_1 [aria-label^='Insert link'] button not found` in local env when I run tests even for branch `5`. 
But there is another issue with limited list of available link type in WIZIWIG.
![multiple-wysiwyg-configs feature_35](https://github.com/silverstripe/silverstripe-elemental/assets/87288324/5142b07c-c79b-4671-bf1c-eb766b31b960)

Reply:
> Without these changes I get  Element #Form_ElementForm_1 [aria-label^='Insert link'] button not found in local env when I run tests even for branch 5.

That would be resolved after merging these changes first into `5.1` (to resolve the failure there) and then merging _up_ to `5`. It's not actually related to the versioned badges work at all.

> But there is another issue with limited list of available link types in WIZIWIG.

That's also separate to this issue and should be raised and resolved separately (but is probably just that asset-admin and cms haven't merged up yet, if I had to guess)

Reply:
Updated